### PR TITLE
Fix/177/open testnet on edit

### PR DESCRIPTION
**Pull request by SanderBuK**

### CODE REVIEW

**Review by thomasdal — APPROVED** [review]

File: src/ts/components/pages/EditProfile/EditProfile.tsx [hunk 1]
```diff
@@ -122,12 +122,18 @@ class UnwrappedEditProfile extends React.PureComponent<EditProfileProps, EditPro
      */
     public componentDidMount(): void {
         const { store } = this.props;
+
         console.log("hi")
         console.log(store.user);
```
Comment (nimrossum) [inline src/ts/components/pages/EditProfile/EditProfile.tsx]: Can these log statements be removed?

File: src/ts/components/pages/EditProfile/EditProfile.tsx [hunk 2]
```diff
@@ -145,7 +151,7 @@ class UnwrappedEditProfile extends React.PureComponent<EditProfileProps, EditPro
                 description: store.user.description,
                 profilePicture: undefined,
                 wallet: isProducerUser(store.user) ? store.user.wallet : "",
-                pairingLink: isProducerUser(store.user) ? store.user.pairingLink : "",
+                pairingLink: isProducerUser(store.user) ? `${process.env.REACT_APP_OBYTE_PROTOCOL}:${process.env.REACT_APP_OBYTE_PAIRING_CODE}#` + (store.user.pairingLink + "").split("#", 2)[1] : "",
```
Comment (nimrossum) [inline src/ts/components/pages/EditProfile/EditProfile.tsx:153]: This seems quite hacky, are there plans on improving this in the back log?

**Review by nimrossum — CHANGES_REQUESTED** [review]

File: src/ts/components/pages/EditProfile/EditProfile.tsx (hunk 2 above)
Comment (SanderBuK) [reply src/ts/components/pages/EditProfile/EditProfile.tsx:153]: Possibly. The optimal solution would be to change the pairingLink all the way in the backend, but Casper has given the greenlight for a sorta hacky, string manipulation solution for now.

Comment (frederikfink) [reply src/ts/components/pages/EditProfile/EditProfile.tsx:153]: I think this is good enough for now :)

**Review by frederikfink — APPROVED** [review]

**Review by emiljapelt — APPROVED** [review]

**Review by nimrossum — APPROVED** [review]

**Review by adamnegaard — APPROVED** [review]
Lil hacky, but looks like it will work for now

**Pull request merged**